app/jobs/send_update_profile_reminder_mail_job.rb

```diff
@@ -0,0 +1,19 @@
+class SendUpdateProfileReminderMailJob < CronJob
+  self.cron_expression = '0 3 * * 7'
+
+  def perform
+    Person.find_each do |person|
+      company = Company.find(person.company_id)
+      if person.reminder_mails_active && company.reminder_mails_active && needs_reminder?(person)
+        ReminderMailer.update_user_reminder_email(person).deliver_now!
+      end
+    end
+  end
+
+  private
+
+  def needs_reminder?(person)
```

Comment:
So you are gonna send them a mail every week? :grimacing: 
Please don't if you don't want people to get mad at us... there should be some form of cooldown for each person :thinking: 
Or maybe only send it it in the week where the profile hasn't been updated for a year.
`.where(updated_at: 1.year.ago..1.year.ago + 7.days)

Also sending it after a year seems a tad late... Maybe send one after 6 months as well.